src/schemas/addToCart-validate.ts

```diff
@@ -55,6 +55,9 @@ export default {
                 price: {
                   optional: true
                 },
+                currency: {
+                  optional: true
```

Comment:
Since we are going to limit the currency to 3 characters in the back end validation, we should add that here too, to make it easier for devs to implement. Otherwise, they'll send a beacon and then not see it in our system if it uses currency like "United States Dollar".

This change should be made for all "validation" schemas for every schema containing "currency".